lib/db/database_loader.rb

```diff
@@ -20,12 +20,22 @@ def add_role(filename)
     filename
   end
 
+  # def drop_and_recreate_database
+  #   ENV['DISABLE_DATABASE_ENVIRONMENT_CHECK'] = '1'
+  #   Rake::Task['db:drop'].invoke
+  #   puts "Database dropped"
+  #   Rake::Task['db:create'].invoke
+  #   puts "Database created"
+  # end
+
   def drop_and_recreate_database
+    # The following way for resetting the db has no need to stop the app server instances 
+    # but after the restoring, you may need to restart the instances in order to clear up the cache.
     ENV['DISABLE_DATABASE_ENVIRONMENT_CHECK'] = '1'
```

Comment:
Sorry, haven't taken the time to look into this, but is there any way this can be run against production?

Reply:
No chance,  unless someone change my code as at very beginning,  I will check the env variable, if prod won't run, but you do have point, actually I don't need this line any more, as I don't drop/reset DB via rake task, let me remove it